### PR TITLE
SNMP enabling and disabling support for Dell iDRAC8 and 9

### PR DESCRIPTION
**Pull request by L1Cafe**
### Checklist
- [ ] Tests added
- [x] Similar commits squashed

### The HW vendor this change applies to (if applicable)

Dell

### The BMC firmware and/or BIOS versions that this change applies to (if applicable)

iDRAC 8 and 9

## Description for changelog/release notes

```
Added initial support for enabling and disabling SNMP on iDRAC 8 and 9
```

### CODE REVIEW

**Comment by L1Cafe** [comment]
I see the linter is complaining about a few things, will fix now

**Comment by L1Cafe** [comment]
@joelrebel please check.

The tests are non-trivial, but these changes were tested in prod machines iDRAC8 and 9

File: providers/dell/idrac8/configure.go [hunk 1]
```diff
@@ -740,6 +741,30 @@ func (i *IDrac8) Network(cfg *cfgresources.Network) (reset bool, err error) {
 	}
 
 	i.log.V(1).Info("Network config parameters applied.", "IP", i.ip, "HardwareType", i.HardwareType())
+
+	// SNMP section
```
Comment (joelrebel) [inline providers/dell/idrac8/configure.go:745]: I suggest this is added into its own function `func (i *IDrac9) SetSNMP(enable bool) error`

**Review by joelrebel — COMMENTED** [review]
left a suggestion, otherwise looks good to me.

File: providers/dell/idrac8/configure.go (hunk 1 above)
Comment (L1Cafe) [reply providers/dell/idrac8/configure.go:745]: Maybe better to make it private?

`func (i *IDrac9) setSNMP(enable bool) error` (lowercase s)

What do you think?

**Comment by L1Cafe** [comment]
Please check @joelrebel

**Review by joelrebel — APPROVED** [review]
LGTM, thanks!

**Pull request merged**